pennylane/io.py

```diff
@@ -407,6 +407,61 @@ def from_qiskit_op(qiskit_op, params=None, wires=None):
         raise RuntimeError(_MISSING_QISKIT_PLUGIN_MESSAGE) from e
 
 
+def from_qiskit_noise(noise_model, **kwargs):
+    """Converts a Qiskit `NoiseModel <https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.noise.NoiseModel.html>`__
+    into a PennyLane :class:`~.NoiseModel`.
```

Comment:
```suggestion
    into a PennyLane :class:`~.NoiseModel`. 
    
    .. note::
    
        Qiskit noise models are converted into an equivalent series of :class:`~.QubitChannel` operators.
```

Thought it would be a good idea to state this right at the start so that the user won't be surprised. @trbromley thoughts?

Reply:
I have included this in the notes we have for the function.

Reply:
IIRC the note was originally there and I suggested merging both notes into one. You can separate them again if it works better.